bench-tps/src/cli.rs

```diff
@@ -382,6 +393,27 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(is_keypair)
                 .help("File containing the node identity (keypair) of a validator with active stake. This allows communicating with network using staked connection"),
         )
+        .arg(
+            Arg::with_name("alt_instruction_program_id")
+                .long("alt-instruction-program-id")
+                .value_name("ALT_INSTRUCTION_PROGRAM_ID")
+                .takes_value(true)
+                .requires("alt_instruction_load_accounts_count")
+                .validator(is_pubkey)
+                .help("If program id of deployed SBF progrem, for example `noop`, is provided; bench transfer transactions \
+                       will include an additional instruction to use the program to load number of accounts from an \
+                       address lookup table account."
+                )
+        )
+        .arg(
+            Arg::with_name("alt_instruction_load_accounts_count")
+                .long("alt-instruction-load-accounts-count")
+                .value_name("ALT_INSTRUCTION_LOAD_ACCOUNTS_COUNT")
+                .takes_value(true)
+                .requires("alt_instruction_program_id")
+                .validator(|n| is_within_range(n, 0..=256))
```

Comment:
is 0 actually a valid number? i.e. is it valid for a transaction to have an alt-lookup without any indexes